src/integrations/third-party/elementor.php

```diff
@@ -468,7 +458,7 @@ public function enqueue() {
 			],
 			'dismissedAlerts'           => $dismissed_alerts,
 			'webinarIntroElementorUrl'  => WPSEO_Shortlinker::get( 'https://yoa.st/webinar-intro-elementor' ),
-			'blackFridayBlockEditorUrl' => ( $this->promotion_manager->is( 'black_friday_2023_checklist' ) ) ? WPSEO_Shortlinker::get( 'https://yoa.st/black-friday-checklist' ) : '',
+			'currentPromotions'         => YoastSEO()->classes->get( Promotion_Manager::class )->get_current_promotions(),
```

Comment:
I think injecting here it the right choice, since we are in the /src directory. I would prefer to not use the services API when we don't have to. Since it is not a best practise to use the DI container directly. It should only happen in the worst case scenario.

Reply:
Done 👍